.github/workflows/changelog-verify.yml

```diff
@@ -46,7 +46,11 @@ jobs:
         # Set outputs using the command.
         run: |
           bash ./.githooks/pre-receive "${{needs.changedfiles.outputs.all}}" 2>&1 | tee output.txt
-          echo "all=$(cat output.txt | jq -sR .)" >> $GITHUB_OUTPUT
+          VALIDATION_OUTPUT=$(cat output.txt)
+          echo "all=$(echo "$VALIDATION_OUTPUT" | jq -sR .)" >> $GITHUB_OUTPUT
+          if [[ "$VALIDATION_OUTPUT" == *"failed"* ]]; then 
+            exit 1
+          fi
```

Comment:
I'm confused why this is showing up in the diff. GitHub simultaneously claims it is present in main: https://github.com/pokt-network/pocket/blob/main/.github/workflows/changelog-verify.yml#L49